docs/ecctl-getting-started.asciidoc

```diff
@@ -252,47 +252,292 @@ https://golang.org/pkg/text/template/[`text/templates`] on demand.
+            "version": "7.6.0" <5>
+          },
+          "deployment_template": {
+            "id": "gcp-io-optimized" <6>
+          }
+        }
+      }
+    ],
+    "kibana": [
+      {
+        "region": "gcp-us-central1",
+        "elasticsearch_cluster_ref_id": "main-elasticsearch",
+        "ref_id": "main-kibana",
+        "plan": {
+          "cluster_topology": [
+            {
+              "instance_configuration_id": "gcp.kibana.1",
+              "zone_count": 1, <7>
+              "size": {
+                "resource": "memory",
+                "value": 1024 <8>
+              }
+            }
+          ],
+          "kibana": {
+            "version": "7.6.0" <9>
+          }
+        }
+      }
+    ],
+    "apm": [
+      {
+        "region": "gcp-us-central1",
+        "elasticsearch_cluster_ref_id": "main-elasticsearch",
+        "ref_id": "main-apm",
+        "plan": {
+          "cluster_topology": [
+            {
+              "instance_configuration_id": "gcp.apm.1",
+              "zone_count": 1, <10>
+              "size": {
+                "resource": "memory",
+                "value": 512 <11>
+              }
+            }
+          ],
+          "apm": {
+            "version": "7.6.0" <12>
+          }
+        }
+      }
+    ]
+  }
+}
+--
+<1> The region for the Elasticsearch cluster
+<2> Instance configuration ID
+<3> The number of availability zones for the Elasticsearch cluster
+<4> The amount of memory allocated for each Elasticsearch node
+<5> The version of the Elasticsearch cluster
+<6> The template on which to base the deployment
+<7> The number of availability zones for Kibana
+<8> The amount of memory allocated for Kibana
+<9> The version of the Kibana instance
+<10> The number of availability zones for APM
+<11> The amount of memory allocated for APM
+<12> The version of the APM instance
+
+This JSON contains the settings for a highly available Elasticsearch cluster deployed across two availability zones, a single instance of Kibana, and a single APM server.
+
+Run the <<ecctl_deployment_create,{p} deployment create>> command with `create-deployment.json` as a parameter. For this and other commands, you can add an optional `--track` parameter to monitor the progress.
+
+[source, sh]
+--
+ecctl deployment create -f create-deployment.json
+--
+
+[source, sh]
 --
-ecctl deployment create --file ./deployment.json --name=my-new-deployment --version=7.4.1
 {
   "created": true,
-  "id": "c44ad3fa558a237d0c17d0f4273801df",
-  "name": "my example cluster",
+  "id": "7229888e7bf8350c7e4d07d7374171c0",
+  "name": "My first ecctl deployment",
   "resources": [
     {
-      "cloud_id": "my_elasticsearch_cluster:REDACTED",
+      "cloud_id": "My_first_ecctl_deployment:dXMtY2VudHJhbDEuZ2NwLmZvdW5kaXQubm8kYjFlZWVjOGQ0YWVlNGY3ZDgxNTM2Zjc1ZjZhN2Y1MDgkM2ViZTAzNmI0NDhkNDc3Y2E2ZTJjZTQ5NmE4ZDQ5ODA=",
       "credentials": {
-        "password": "REDACTED",
+        "password": "qIJuMbL8BFfgoNq5bpjQhDRC",
```

Comment:
I realise these are fake credentials, but should we write them out like this? Is it not better to use `REDACTED` ?

Reply:
Definitely!  👍 
It's fixed now.